src/Portal/docs/New-AzPortalDashboard.md

```diff
@@ -0,0 +1,266 @@
+---
+external help file:
+Module Name: Az.Portal
+online version: https://docs.microsoft.com/en-us/powershell/module/az.portal/new-azportaldashboard
+schema: 2.0.0
+---
+
+# New-AzPortalDashboard
+
+## SYNOPSIS
+Creates or updates a Dashboard.
+
+## SYNTAX
+
+### CreateExpanded (Default)
+```
+New-AzPortalDashboard -Name <String> -ResourceGroupName <String> -Location <String> [-SubscriptionId <String>]
+ [-Lens <Hashtable>] [-Metadata <Hashtable>] [-Tag <Hashtable>] [-DefaultProfile <PSObject>] [-Confirm]
+ [-WhatIf] [<CommonParameters>]
+```
+
+### Create
```

Comment:
"unexpanded" parameter set should be removed or hidden.

Reply:
In this case, per the design review, we decided to leave the unexpanded parameter set, as the pieces of the Dashboard are all complex types, and may not make as much sence to the user as the 'Dashboard' object